secant/Features/BackupFlow/RecoveryPhraseValidation.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  RecoveryPhraseValidation.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/29/21.
+//
+
+import Foundation
+import ComposableArchitecture
+import SwiftUI
+
+typealias RecoveryPhraseValidationStore = Store<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+typealias RecoveryPhraseValidationViewStore = ViewStore<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+
+/// Represents the completion of a group of recovery words by the addition of one word into the given group
+struct ValidationWord: Equatable {
+    var groupIndex: Int
+    var word: String
+}
+
+struct RecoveryPhraseValidationState: Equatable {
+    enum Step: Equatable {
+        case initial
+        case incomplete
+        case complete
+    }
+
+    enum Route: Equatable, CaseIterable {
+        case success
+        case failure
+    }
+
+    static let wordGroupSize = 6
+    static let phraseChunks = 4
+
+    var phrase: RecoveryPhrase
+    var missingIndices: [Int]
+    var missingWordChips: [PhraseChip.Kind]
+    var validationWords: [ValidationWord]
+    var route: Route?
+    var step: Step {
```

Comment:
Can we add a space between the stored properties and computed properties?  It helps to easily see what this type consists of.

```suggestion
    var route: Route?
    
    var step: Step {
```